### DISCUSSION TITLE
安装1.10.0版本报错

### DISCUSSION THREAD

**Discussion by JaySem** [hhyo/Archery#2360] [Q&A]
### 重现步骤

1.安装好docker
2.下载好docker compose
sudo curl -L "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose

sudo chmod +x /usr/local/bin/docker-compose

3./usr/local/bin/docker-compose -f /data/Archery-1.10.0/src/docker-compose/docker-compose.yml  up -d 
报错 ERROR: missing signature key

### 预期外的结果

第3步报错：
Trying to pull repository docker.io/library/mysql ... 
ERROR: missing signature key

### 版本

1.10.0

### 部署方式

Docker

**Comment by finovy** [comment]
我是用Ubuntu22.04 apt安装docker的，没有问题
